pallets/capacity/src/lib.rs

```diff
@@ -828,9 +824,12 @@ impl<T: Config> Pallet<T> {
 		staker: &T::AccountId,
 		proposed_amount: BalanceOf<T>,
 	) -> BalanceOf<T> {
-		let account_balance =
-			T::Currency::reducible_balance(staker, Preservation::Preserve, Fortitude::Polite);
-		let stakable_amount = account_balance.saturating_sub(T::MinimumTokenBalance::get());
+		let freezable_balance = T::Currency::balance_freezable(staker);
+		let current_staking_balance =
+			StakingAccountLedger::<T>::get(staker).unwrap_or_default().active;
```

Comment:
Does this increase the db reads for the call? I would think not as surely it is already in there somewhere right?

Reply:
I ran the benchmarks locally and it does not increase reads.